locale/zh/messages.json

```diff
@@ -918,9 +920,9 @@
   "The CPUs offered": "已提供 CPU",
   "The GPUs offered": "已提供 GPU",
   "The Mesos master process orchestrates agent tasks.": "Mesos 管理节点进程编排代理任务。",
-  "The command that is executed. This value is wrapped by Mesos via `/bin/sh -c job.cmd`. Either `cmd` or `args` must be supplied. It is invalid to supply both `cmd` and `args` in the same job.": "",
   "The UI rollback has failed due to an error.": "",
   "The UI upgrade has failed due to an error.": "",
+  "The command that is executed. This value is wrapped by Mesos via `/bin/sh -c job.cmd`. Either `cmd` or `args` must be supplied. It is invalid to supply both `cmd` and `args` in the same job.": "",
```

Comment:
I have no idea why it would remove this message and then re-insert it a few lines below 🤷‍♀️

Reply:
I think lingui has some internal method to match lines with files, like the `line` attribute it had in the other format 🤔